001-quickstart-passwordless/app.csproj

```
@@ -0,0 +1,12 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net6.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="Azure.Identity" Version="1.8.0" />
```

Do you think we should use a flexible version number here? I'm doing it for Microsoft.Azure.Cosmos, but I'm open to debate on the topic.

```suggestion
    <PackageReference Include="Azure.Identity" Version="1.*" />
```